x-pack/platform/plugins/shared/onechat/public/application/mount.tsx

```diff
@@ -26,7 +26,7 @@ export const mountApp = async ({
   services,
 }: {
   core: CoreStart;
-  plugins: OnechatStartDependencies;
+  plugins: OnechatPluginStart;
```

Comment:
I'm confused on that type switch? In `x-pack/platform/plugins/shared/onechat/public/types.ts`, `OnechatPluginStart` is an empty interface?